libs/payments/cart/src/lib/util/isPaymentIntent.ts

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export function isPaymentIntent(intentId: string) {
+  return !!intentId.startsWith('pi_')
+}
```

Comment:
This feels like it could be a type predicate https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates

Reply:
I'm not sure a type predicate would work here, since the function identifies whether the intentId, of type `string`, is a string ID of `SetupIntent` or `PaymentIntent`.

That said, this function should probably be renamed `isPaymentIntentId` so that it's clear its intended for an ID and not the actual Intent object.

Reply:
Sounds good enough to me!